src/ssmDocument/commands/publishDocument.ts

```diff
@@ -182,23 +186,56 @@ export async function updateDocument(
         outputChannel.appendLine(
             localize(
                 'AWS.message.info.ssmDocument.publishDocument.updateSuccess',
-                "Successfully updated SSM Document '{0}'",
+                "Successfully updated Systems Manager Document '{0}'",
                 wizardResponse.name
             )
         )
         if (updateResult.DocumentDescription) {
             outputChannel.appendLine(stringify(updateResult.DocumentDescription))
         }
-        logger.info(`Updated SSM Document successfully ${stringify(updateResult.DocumentDescription)}`)
-        vscode.window.showInformationMessage(`Updated SSM Document successfully`)
+
+        logger.info(`Updated Systems Manager Document successfully ${stringify(updateResult.DocumentDescription)}`)
+        vscode.window.showInformationMessage(`Updated Systems Manager Document successfully`)
         outputChannel.appendLine('')
+
+        const isConfirmed = await showConfirmationMessage(
+            {
+                prompt: localize(
+                    'AWS.ssmDocument.publishDocument.updateVersion.prompt',
+                    'Would you like to make this the default version for {0}?',
+                    wizardResponse.name
+                ),
+                confirm: localize('AWS.ssmDocument.publishDocument.updateVersion.confirm', 'Yes'),
+                cancel: localize('AWS.ssmDocument.publishDocument.updateVersion.cancel', 'No'),
```

Comment:
Nit: Use the generic strings:
```
    "AWS.generic.response.no": "No",
    "AWS.generic.response.yes": "Yes"
```

(admittedly, we have multiple other "Yes/No" strings in the localization files, but it's not a bad idea to avoid another :) )

Reply:
Will use generic strings